streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -542,4 +543,29 @@ public void shouldMergeMultipleStreams() {
         assertEquals(Utils.mkList("A:aa", "B:bb", "C:cc", "D:dd", "E:ee", "F:ff", "G:gg", "H:hh"),
                      processorSupplier.processed);
     }
+
+    @Test
+    public void shouldProcessFromSourcesThatMatchPattern() {
```

Comment:
Can we add a simple test with most basic topology, ie, only a single source pattern and a sink?